src/base-styles/helperClasses/border/focus.css

```diff
@@ -0,0 +1,24 @@
+/*---
+title: Focus Style
+name: focusStyle
+section: Border Helpers
+---
+
+Adds standard focus style to elements
```

Comment:
might be nice to clarify in docs that the class is selecting the `:focus` state. I can see someone wondering why they don't see the focus outline by just adding the class